packages/editor/playground/playground.tsx

```diff
@@ -57,9 +57,17 @@ const BlurHelper: FC = () => {
 
 const DebugConsole: FC<{ hasUnsavedChanges: boolean; content: string }> = ({ hasUnsavedChanges, content }) => {
     return (
-        <div>
-            <p>content: {content}</p>
-            <p>hasUnsavedChanges: {JSON.stringify(hasUnsavedChanges)}</p>
+        <div className="css-t4vd4r" style={{ borderTop: "1px solid gray" }}>
+            <div className="ProseMirror">
```

Comment:
Remove these two `className`. `DebugConsole` is not part of the ProseMirror editor.

Reply:
what would I do if I want to use the style of the editor above. handwriting instead of reusing classes？
PS: I find class`css-t4vd4r` in browser devtools.

Reply:
1. Don't reuse the style of the editor above. They are two different components with different purposes, so they don't have to share the same style.
2. `css-t4vd4r` is auto-generated in runtime, so this name could change.